test/app.spec.js

```diff
@@ -97,6 +97,22 @@ describe("App", () => {
   });
 })
 
+describe("Views", () => {
+  it("allows inline styles as object with properties in camelCase", () => {
```

Comment:
@danigb Can you explain this test? HyperApp is not really doing anything here, since we're calling setattribute, the DOM expects the property to be camelCase anyway.